opencti-platform/opencti-front/src/private/components/settings/users/User.tsx

```diff
@@ -326,6 +328,12 @@ const User: FunctionComponent<UserProps> = ({ userData, refetch }) => {
     .sort(
       (a: Session, b: Session) => timestamp(a?.created) - timestamp(b?.created),
     );
+
+  let accountExpireDate = fldt(user.account_lock_after_date);
+  if (accountExpireDate === '-') {
+    accountExpireDate = 'None';
```

Comment:
We decided that empty fields should be displayed as '-' (and not 'None') (to be homogene with the rest of the platform).

Reply:
@Archidoit - Committing this fix now.